### DISCUSSION TITLE
Forge Version?

### DISCUSSION THREAD

**Discussion by GHCreeper** [MrKacafirekCZ/doggo-mod-overhauled#48] [Q&A]
I'm okay at making minecraft mods for forge, and since your mod doesn't have a forge version, could I be allowed to make a forge port of this mod? I'd obviously give you credit.

**Comment by MrKacafirekCZ** [comment]
Hello, I believe that creating a completely separate repository, separate mod page, separate discussions would only lead to confusion for the regular player using the Fabric or Forge version. Issues and questions might end up on the wrong version page and it would only add unnecessary problems in the long run.

**That being said however!** I'm not against the idea of creating a forge port. Instead of creating a completely separate repository/mod page, you could create a pull request with a fully functioning Forge version of this mod. Then there would be one repository, one mod page and one place to discuss new features and potential bugs and I also believe that it would make managing both versions much easier as they could get updates at the same time.

**Reply by GHCreeper** [reply]
Ok. I'll try creating a pull request!

EDIT: One problem, how do I create a pull request?

**Reply by MrKacafirekCZ** [reply]
1. Fork this repository on GitHub.
2. Clone your version of this repository (the forked version) to your local machine.
3. Make all necessary changes to make a fully functioning Forge version.
  - Keep in mind that you also need to keep the Fabric version at best completely untouched. Currently I have no idea what works the best so an easy solution could be to have a folder named "Fabric" and inside all files and folders that make a Fabric version working and a folder named "Forge" and inside all files and folders that make a Forge version working.
4. Once done with changes, commit and push to your forked repository.
5. Create a pull request.

**Reply by GHCreeper** [reply]
I'm confused. Do you want me to create a forge version or not?
